recipe/meta.yaml

```diff
@@ -16,28 +16,29 @@ build:
 requirements:
   host:
     # Drop support for python 3.6 https://github.com/pydata/xarray/pull/4720
-    - python >=3.7
+    - python
     - pip
-    - setuptools >=42
+    - setuptools
     - setuptools_scm_git_archive
-    - setuptools_scm >=3.4
```

Comment:
There should be `- setuptools_scm >=3.4`, see https://github.com/pydata/xarray/blob/12a216d632111fc6be39cdaa98335a9bb22f2c5d/pyproject.toml#L5